### ISSUE TITLE
Running the MDNet demo on CPU

### ISSUE THREAD

**Issue by nitish11** [HyeonseobNam/MDNet#2]
Hi,

How to run the model on CPU?

**Comment by sysu-zjw** [comment]
Hi,I use MATLAB 2016a and VS 2015 C++ ,and have switched "opts.useGpu"  to false.
But I got the same error as @nitish11 did.

**Comment by HyeonseobNam** [comment]
@nitish11 @sysu-zjw 
Hi, 
Have you tried removing "vl_simplenn_move" lines? (line 89-90 in mdnet_init)
The model parameters are stored in the matlab format of "gpuArray", and I did not tested it on the environment where no gpu is installed.
If you have a gpu installed, you may need to check the CUDA version.

**Comment by sysu-zjw** [comment]
@HyeonseobNam 
I have tried your advice. However, it didn't work. I think it may relate to the version of matlab.
I will try the GPU version. Thanks for sharing! :)

**Comment by salmanmaq** [comment]
A straight-forward solution would be if you can provide us the model .mat files in the CPU format i.e. they contain normal MatLab arrays, not gpuArrays. That would resolve this issue.

**Comment by HyeonseobNam** [comment]
@nitish11 @sysu-zjw @salmanmaq 
For now you can download the model files in the CPU format from:
https://drive.google.com/open?id=0B4G5mwcHYoSuMXo2eUpkbkpTWGc
Thanks :)

**Comment by salmanmaq** [comment]
Thank you so much. Just what I have been waiting for. :)
I'll test it out and update and comment.

**Comment by SoulmateB** [comment]
@salmanmaq 
Hi,have you got the the model .mat files in the CPU format? I have the same problem when I run the demo on CPU, could you tell me how to solve it ? Thank you !!

**Comment by SoulmateB** [comment]
@HyeonseobNam 
How to run the model on CPU? I have the same problem when I run the demo on CPU.....Thank you !!

**Comment by salmanmaq** [comment]
@SoulmateB I did this a few months back. The files HyeonseobNam provided in the Drive worked fine for me. Did you try them out?

**Comment by SoulmateB** [comment]
No,I can't download the files,could you provide me the files? Thank you !!

**Comment by salmanmaq** [comment]
I just checked and the link works for me. Do you encounter any particular error with the link?

**Comment by afunTW** [comment]
I downloaded the model in CPU format and apply it in demo_tracking.m

Although the console show the message 'Starting parallel pool (parpool) using the 'local' profile ... connected to 4 workers', but it only use one CPU computing resource in fact, it there anyone get the same situation?

**Comment by afunTW** [comment]
I downloaded the model in CPU format and applied it in demo_tracking.m
My Parallel Computing Toolbox version is 6.9 in Matlab R2016b

However, when I run the demo_tracking.m, the program will idle over 30 minutes and leads the parallel pool close. Am I in the right way to run this program in CPU mode?
